packages/terra-application-docs/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 * Changed
   * Minor dependency version bump
+  * Remove references to github user content.
```

Comment:
i on purpose added this here because once the main build is fixed this will get released.